src/baseline_rule.py

```diff
@@ -128,15 +145,102 @@ def get_port_array(self):
 
         return ports_array
 
+    def get_port_array_calico(self):
+        if not self.port_min:
+            return [], self.protocol
+        ports_array = []
+        for port in range(self.port_min, self.port_max + 1):
+            ports_array.append(port)
+
```

Comment:
Calico and K8s have support for port ranges - should we use it? It could be more efficient

Reply:
done